tool/plugin/lib/plugin.dart

```diff
@@ -309,13 +310,87 @@ void _copyResources(Directory from, Directory to) {
   }
 }
 
+class AntBuildCommand extends BuildCommand {
+  AntBuildCommand(BuildCommandRunner runner) : super(runner, 'build');
+
+  Future<int> externalBuildCommand(BuildSpec spec) async {
+    return runner.javac2(spec);
+  }
+
+  Future<int> savePluginArtifact(BuildSpec spec, String version) async {
+    int result;
+
+    // create the jars
+    createDir('build/flutter-intellij/lib');
+    result = await jar(
+        'build/classes', 'build/flutter-intellij/lib/flutter-intellij.jar');
```

Comment:
nit: trailing comma after last param. There are several places in this file where this should also be applied. Sorry in advance for spamming with comments :)

Reply:
Done